streamx-console/streamx-console-webapp/src/views/flink/app/AddEdit.js

```diff
@@ -269,20 +269,29 @@ export function bigScreenClose(vue) {
 export function applyPom(vue) {
   const controller = vue.controller
   const pom = controller.pom.value
+  const versionId = vue.versionId
+  const scalaVersion = vue.flinkEnvs.find(v => v.id === versionId).scalaVersion
```

Comment:
In this way, it will bring another problem: when switching the flick version at the top, you need to recheck the dependencies in the POM

____
这样一来的话,会带来另外一个问题需要注意: 这里已经和用户选择的flinkVersion绑定起来了,一旦用户选择其他的flinkVersion, 则要重新走一遍检查pom依赖